code/outlierfunction.py

```diff
@@ -0,0 +1,19 @@
+
+# find outliers based on DVARS and FD
+
+def outlier(file_name, save_name, bound):
+    # load data
+    data = np.loadtxt(file_name)
+    outlier = []
+    # set outlier values to 0
+    for i in data:
+        if i <= bound:
+            outlier.append(0)
+        else:
+            outlier.append(i)
+    # find outlier indices
+    outlier_indices = np.nonzero(outlier)
+    # write file to store outlier indices
```

Comment:
Do we these three lines below inside the function? do you think it may be better to just output indicies and actually save in the script (create an array that stores the list of indicies from each run and save this array as txt), or include a boolean indicator in parameter whether to save or not.